src/User.ts

```diff
@@ -12,6 +12,27 @@ export interface UserProfile {
     azp?: string;
     at_hash?: string;
     auth_time?: number;
+
+    // Other standard claims - https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
```

Comment:
It would be helpful to include this link in the TSDoc for the class definition above (before the `@public` hint). Something like

```typescript
/**
  * @see https://openid.net/specs/openid-connect-core-1_0.html#StandardClaims
  *
  * @public
  */
export interface UserProfile {
```

Reply:
See comment below